### DISCUSSION TITLE
spanish keyboard issue

### DISCUSSION THREAD

**Discussion by 0-andy-0** [ghostty-org/ghostty#3474] [Feature Requests, Ideas] [answered]
On my keyboard 
   shift+, is ;
and
  shift+. is :

According to terminal inspector, ghostty gets inconsistent 
  shift+semicolon
and
  shift+period
respectively.

Why not shift+comma ?

Upshot: shift+control+comma (the shortcut for reloading config) cannot be issue from the keyboard

**Comment by polirritmico** [comment]
Same issue here. As result the default `ctrl+shift+comma` for the `reload_config` action doesn't work out of the box and prints `;5u` (bash shell). Indeed to fix it `semicolon` needs to be used:

```
# fix reload_config default keymap
keybind = ctrl+shift+semicolon=reload_config
```

**Reply by 0-andy-0** [reply]
For completeness:
 
| Shift+KEY | Interpreted as Shift+ |
|--------|--------|
| , | semicolon|
| 0 | equal |
| 7| slash |
| < | > |
| ñ| Ñ |
| ç| Ç |
| ¡ | ¿ |
| º| ª |

Terminal inspector seems to report correctly all other keys when shifted, as shift+key.  So Shift+six, Shift+nine, but not Shift+seven or Shift+zero.  

A possibly-related (but could be mixed up with deadkey handling) oddity:  the key sequence   AltGr+ '  which produces {  is reported as the following unbalanced events:

- Press: right_alt
- Press: {
- Release: apostrophe
- Release: right_alt

**Comment by slovdahl** [comment, marked as answer]
Cross-linking, I wrote https://github.com/ghostty-org/ghostty/discussions/4427 before I found this.
